cql3/query_processor.cc

```diff
@@ -103,7 +103,7 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, qu
     qp_group.push_back(sm::make_derive(
```

Comment:
Can't comment on the commit description, so will comment here: Please split the commit description into two parts. A single line summary (or subject line) that is preferably 72 chars or shorter (not always possible), followed by the the more elaborate description of what the commit does, separated from the subject line with an empty line.

Also please move the cosmetic changes into a separate commit, like @kbr- requested.